### DISCUSSION TITLE
Editing the text for custom secondary file formats

### DISCUSSION THREAD

**Discussion by aghaynes** [quarto-dev/quarto-cli#11536] [Q&A] [answered]
### Description

It is possible to output multiple formats simultaneously by specifying them as defaults. It's also possible to customise the text thats shown for under under "other links". However, when extensions are used, the extension type seems to be appended to the text entered. Here I've used the `quarto-letter` extension:

````
quarto use template mcanouil/quarto-letter
````

```` qmd
---
title: "Untitled"
format: 
  html: default
  typst: default
  letter-pdf: default
format-links:
  - html
  - format: typst
    text: PDF
    icon: file-pdf
    target: "_blank"
  - format: letter-pdf
    text: PDF 2
    icon: file-pdf
    target: "_blank"
---

## Quarto

Quarto enables you to weave together content and executable code into a finished document. To learn more about Quarto see <https://quarto.org>.
```` 

That produces 
![image](https://github.com/user-attachments/assets/b8962e0c-9e2e-45d4-a206-07bbdf3ff208)

Note that quarto has appended "(letter)" to PDF 2. Is it possible to override that behaviour? If I had wanted it to say "PDF 2 (letter)" I would have written that in the text argument... 

Thanks!

**Comment by mcanouil** [comment, marked as answer]
Unfortunately, that's not currently possible:

- https://github.com/quarto-dev/quarto-cli/issues/10273

**Reply by aghaynes** [reply]
exactly that! any idea if/when it will be implemented?

**Reply by mcanouil** [reply]
Since 1.6 was just released, no idea when.
it could be part of 1.7 but could be for later than that.

**Reply by aghaynes** [reply]
from my side, if it could be part of 1.7, that would be great. There was another comment that it was also stopping it being used in production too...

**Reply by mcanouil** [reply]
You could hack your way with JavaScript if really needed.
